hunting/__main__.py

```diff
@@ -51,6 +52,26 @@ def generate_markdown(path: Path = None):
     # After processing, update the index
     markdown_generator.update_index_md()
 
+@hunting.command('generate-json')
+@click.argument('path', required=False)
```

Comment:
```
from pathlib import Path

...

@click.argument("path", type=click.Path(dir_okay=True, path_type=Path, exists=True))
```
would ensure the argument `path` has the correct type, so there will be no need for forced conversion below:
```
path = Path(path)
```